tests/app/dao/test_ft_billing_dao.py

```diff
@@ -606,62 +606,111 @@ def test_fetch_sms_billing_for_all_services_with_remainder(notify_db_session):
 
 
 def test_fetch_sms_billing_for_all_services_without_an_organisation_appears(notify_db_session):
-    org, org_2, service, service_2, service_3, service_sms_only, \
-        org_with_emails, service_with_emails = set_up_usage_data(datetime(2019, 5, 1))
+    setup = set_up_usage_data(datetime(2019, 5, 1))
     results = fetch_sms_billing_for_all_services(datetime(2019, 5, 1), datetime(2019, 5, 31))
 
     assert len(results) == 2
     # organisation_name, organisation_id, service_name, service_id, free_sms_fragment_limit,
     # sms_rate, sms_remainder, sms_billable_units, chargeable_billable_units, sms_cost
-    assert results[0] == (org.name, org.id, service.name, service.id, 10, Decimal('0.11'), 8, 3, 0, Decimal('0'))
-    assert results[1] == (None, None, service_sms_only.name, service_sms_only.id, 10, Decimal('0.11'),
-                          0, 3, 3, Decimal('0.33'))
+    assert results[0] == (
+        setup["org_1"].name,
+        setup["org_1"].id,
+        setup["service_1_sms_and_letter"].name,
+        setup["service_1_sms_and_letter"].id,
+        10, Decimal('0.11'), 8, 3, 0, Decimal('0'))
+    assert results[1] == (
+        None,
+        None,
+        setup["service_with_sms_without_org"].name,
+        setup["service_with_sms_without_org"].id,
+        10, Decimal('0.11'), 0, 3, 3, Decimal('0.33')
+    )
 
 
 def test_fetch_letter_costs_for_all_services(notify_db_session):
-    org, org_2, service, service_2, service_3, service_sms_only, \
-        org_with_emails, service_with_emails = set_up_usage_data(datetime(2019, 6, 1))
+    setup = set_up_usage_data(datetime(2019, 6, 1))
 
     results = fetch_letter_costs_for_all_services(datetime(2019, 6, 1), datetime(2019, 9, 30))
 
     assert len(results) == 3
-    assert results[0] == (org.name, org.id, service.name, service.id, Decimal('3.40'))
-    assert results[1] == (org_2.name, org_2.id, service_2.name, service_2.id, Decimal('14.00'))
-    assert results[2] == (None, None, service_3.name, service_3.id, Decimal('24.45'))
+    assert results[0] == (
+        setup["org_1"].name, setup["org_1"].id,
+        setup["service_1_sms_and_letter"].name, setup["service_1_sms_and_letter"].id,
+        Decimal('3.40')
+    )
+    assert results[1] == (
+        setup["org_for_service_with_letters"].name, setup["org_for_service_with_letters"].id,
+        setup["service_with_letters"].name, setup["service_with_letters"].id,
```

Comment:
I think it was named wrong then, service_with_emails should have only emails, this service definitely has letters.

Reply:
Now I remember when changing this, I saw names for a few services were not correct in the fixture, so I changed them to what they really are, and grouped billing records by each service, so it is easier to see what that service does.

Feel free to go and check in the fixture ☺️

Reply:
Ah yes, that's really confusing how the names are totally different in the two places. I can see what you mean now, so I'll go through and resolve the other comments. Thanks for explaining the difference.

I did find this commit quite hard to review, due to the number of variables being changed. As a future suggestion (although hopefully we don't have other code like this!), I think it would have been easier to review as two commits: one to make the variable names consistent everywhere; and then anther to change them to a hash.